build.gradle

```diff
@@ -254,7 +254,7 @@ project(':core') {
 
   dependencies {
     compile project(':clients')
-    compile "$slf4jlog4j"
+    compile project(':log4j-appender')
```

Comment:
Filed KAFKA-2716, we can discuss there, so that discussion is not lost.

On Fri, Oct 30, 2015 at 5:34 PM, Ismael Juma <notifications@github.com>
wrote:

> In build.gradle
> <https://github.com/apache/kafka/pull/395#discussion_r43565100>:
>
> > @@ -254,7 +254,7 @@ project(':core') {
> >
> >    dependencies {
> >      compile project(':clients')
> > -    compile "$slf4jlog4j"
> > +    compile project(':log4j-appender')
>
> I'm still interested in understanding why core needs this as a dep of
> course.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/kafka/pull/395/files#r43565100>.
>



-- 

Regards,
Ashish